tests/test_finite_diff.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import pytest
+
+import pennylane as qml
+
+
+x1 = 0.5
+y1 = -0.3
+
+x2 = np.array([0.5, -0.1975])
+y2 = -0.3
+
+
+@pytest.mark.parametrize(
+    ("x", "y", "argnum", "idx", "delta", "exp_grad"),
```

Comment:
Do you need to parametrize `x` and `y`? :thinking:

Reply:
Probably we can give them explicitly in the list but I think will deteriorate readability of the tests ?